Nazar65/magento2-tasks
# Mass Product Update via csv Implemntation 55811

### Description (*)
Provides yo ability to update products prices, qty, visibility .. for customers via csv file.
### Fixed Issues (if relevant)
N/A
### Manual testing scenarios (*)
1. Go to admin TSG TAB -> click Mass Product Update
2. Download sample file 
3. add file to import 
4. Click Check data
5. Click Import
6. Enjoy.
 
### Contribution checklist (*)
